src/main/java/com/sap/oss/phosphor/fosstars/model/Subject.java

```diff
@@ -71,4 +73,18 @@
 
     return clazz.cast(subject);
   }
+
+  /**
+   * Clones a subject to avoid JSON mapping infinite recursion loop.
+   * The subject is cloned to a different entity.
```

Comment:
This method can be used for various purposes.

```suggestion
   * Clones a subject.
```